cl/simple_pages/templates/contribute.html

```diff
@@ -26,7 +26,7 @@ <h1>Contributing to CourtListener and Free&nbsp;Law&nbsp;Project</h1>
   <p>If you are looking to support our work, the first thing you can do to help is donate to Free Law Project. This allows us to hire more full time, long term staff members. It can be gauche to say so directly, but nothing makes as big a difference as financial support. Please consider donating if you can.
   </p>
   <p>
-    <a href="{% url "donate" %}?referrer=contribute-page" class="btn btn-danger btn-lg">Donate Now</a>
+    <a href="https://donate.free.law/forms/supportflp" class="btn btn-danger btn-lg">Donate Now</a>
```

Comment:
Tweaks, please:

```
  <p>If you are looking to support our work, the first thing you can do to help is donate to Free Law Project or join us as a member. This allows us to hire more full time staff. It can be uncomfortable to say so directly, but nothing makes as big a difference as financial support. Please consider donating or becoming a member if you can.
  </p>
```

And then we'll need the double buttons again as elsewhere.